### ISSUE TITLE
QGIS Configuration: abbreviated command line options not visibile

### ISSUE THREAD

**Issue by agiudiceandrea** [qgis/QGIS-Documentation#7599]
## Description

In the "Command line and environment variables" paragraph of the "QGIS Configuration" chapter, the list of the command line options misses the abbreviated form

![image](https://user-images.githubusercontent.com/16253859/170966732-104fb2c6-4bce-4b63-8e62-e73bdf344a94.png)

while the abbreviated form is present in the code of such page

```
  QGIS is a user friendly Open Source Geographic Information System.
  Usage: /usr/bin/qgis.bin [OPTION] [FILE]
    OPTION:
          [-v, --version]     display version information and exit
          [-s, --snapshot filename]   emit snapshot of loaded datasets to given file
          [-w, --width width] width of snapshot to emit
          [-h, --height height]       height of snapshot to emit
          [-l, --lang language]       use language for interface text (changes existing override)
          [-p, --project projectfile] load the given QGIS project
          [-e, --extent xmin,ymin,xmax,ymax]  set initial map extent
          [-n, --nologo]      hide splash screen
          [-V, --noversioncheck]      don't check for new version of QGIS at startup
          [-P, --noplugins]   don't restore plugins on startup
          [-B, --skipbadlayers]     don't prompt for missing layers
          [-C, --nocustomization]     don't apply GUI customization
          [-z, --customizationfile path]      use the given ini file as GUI customization

```

Page URL: https://docs.qgis.org/3.22/en/docs/user_manual/introduction/qgis_configuration.html#command-line-and-environment-variables

**Comment by DelazJ** [comment]
@agiudiceandrea See https://docs.qgis.org/testing/en/docs/user_manual/introduction/qgis_configuration.html#command-line-and-environment-variables I'm not sure these shortcuts are also available in 3.22...

**Comment by agiudiceandrea** [comment]
Mmmhhh... I didn't notice this inconsistency between versions...

While the CLI help text has been updated only in master and 3.24 with https://github.com/qgis/QGIS/pull/46548 in order to show the short form of the command line options, such short form are actually available and usable in QGIS 3.22 and previous versions (at least since 3.0).
I think https://github.com/qgis/QGIS/pull/46548 and https://github.com/qgis/QGIS/pull/48267 should be backported to 3.22.

Anyway, since the doc says: "qgis --help on the command line, which returns:", then you are right and the 3.22 docs should not be changed until https://github.com/qgis/QGIS/pull/46548 and https://github.com/qgis/QGIS/pull/48267 are backported.
